coderd/database/migrations/000301_system_user.up.sql

```diff
@@ -0,0 +1,51 @@
+ALTER TABLE users
+	ADD COLUMN is_system bool DEFAULT false;
+
+CREATE INDEX user_is_system_idx ON users USING btree (is_system);
+
+COMMENT ON COLUMN users.is_system IS 'Determines if a user is a system user, and therefore cannot login or perform normal actions';
+
+-- TODO: tried using "none" for login type, but the migration produced this error: 'unsafe use of new value "none" of enum type login_type'
+--  	 -> not sure why though? it exists on the login_type enum.
+INSERT INTO users (id, email, username, name, created_at, updated_at, status, rbac_roles, hashed_password, is_system, login_type)
+VALUES ('c42fdf75-3097-471c-8c33-fb52454d81c0', 'prebuilds@system', 'prebuilds', 'Prebuilds Owner', now(), now(),
```

Comment:
This ID will be well-known, and is defined in `coderd/prebuilds/id.go` to be used all over.
We need a static ID for this user so we can identity it.